plugins/generator-1.19.4/forge-1.19.4/templates/overlay.java.ftl

```diff
@@ -129,4 +128,35 @@ package ${package}.client.screens;
 	}
 
 }
+
+<#macro calculatePos anchor_point, comp_x, comp_y>
+	<#if anchor_point == "TOP_LEFT">
+		posX = ${comp_x};
+		posY = ${comp_y};
+	<#elseif anchor_point == "TOP_CENTER">
+		posX = w / 2 + ${comp_x - 213};
+		posY = ${comp_y};
+	<#elseif anchor_point == "TOP_RIGHT">
+		posX = w - (427 - ${comp_x});

```

Comment:
You can optimize a bit the code by doing `(${fixedNumber - ftlVar})`, so in this specific case, it would look like this `posX = w - (${427 - comp_x});`.

Apply this to the other anchor points in both generators